packages/manager/apps/cloud/client/app/dbaas/logs/detail/streams/home/logs-streams-home.html

```diff
@@ -44,7 +44,7 @@ <h3 data-translate="logs_streams_title"></h3>
             <span
                 class="oui-badge"
                 data-translate="{{ $row.info.indexingEnabled ? 'logs_streams_enabled' : 'logs_streams_disabled' }}"
-                data-ng-class="$row.info.indexingEnabled ? 'oui-status_success' : 'oui-status_warning'"
+                data-ng-class="$row.info.indexingEnabled ? 'success' : 'oarning'"
```

Comment:
```suggestion
                data-ng-class="$row.info.indexingEnabled ? 'success' : 'warning'"
```

Reply:
Good catch, but this should have been fixed with `oui-badge_success` and `oui-badge_warning`.
I'll fix that and check what's the reason behind that ;)